civihr_employee_portal/features/civihr_employee_portal_features/civihr_employee_portal_features.features.filter.inc

```diff
@@ -0,0 +1,24 @@
+<?php
+/**
+ * @file
+ * civihr_employee_portal_features.features.filter.inc
+ */
+
+/**
+ * Implements hook_filter_default_formats().
+ */
+function civihr_employee_portal_features_filter_default_formats() {
```

Comment:
could you give me an example of what this is doing and where we are using it? Please provide screenshots to give as much context as you can

Reply:
Sure. Information was added in the description.
PD: this work has some history since here when I tried to find a solution to another bug https://github.com/compucorp/civihr-employee-portal/pull/498